variables.tf

```
@@ -93,6 +93,11 @@ variable "image_id" {
   default     = ""
 }
 
+variable "use_custom_image_id" {
+  description = "if true will use variable image_id to run EKS workers inside autoscaling group"
+  default     = false
```

```suggestion
  default     = "false"
```

TF recommends using strings instead of booleans in input variables

---

ok, also how about the second proposition to fix isue with count cannot be computed in previous comment?